src/EditorFeatures/Core/Host/IStreamingFindReferencesPresenter.cs

```diff
@@ -80,7 +80,9 @@ internal static class IStreamingFindUsagesPresenterExtensions
                 nonExternalItems[0].SourceSpans.Length <= 1)
             {
                 // There was only one location to navigate to.  Just directly go to that location.
-                return nonExternalItems[0].TryNavigateTo(workspace, isPreview: true);
+                // If we're directly going to a location we need to activate the preview to keep focus 
+                // in the correct spot
```

Comment:
the comment doesn't really help me understand this.  "we need to activiate the preview to keep focus in hte correct spot".  why?  what is "the correct spot" here?  why would passing "false" be bad, or what behavior woudl it have?

Reply:
```
                // There was only one location to navigate to.  Just directly go to that location.
                // If we're directly going to a location we need to activate the preview so
                // that focus follows to the new cursor position.
``` 

Does that help clarify?